### ISSUE TITLE
Continue training from checkpoint?

### ISSUE THREAD

**Issue by erjinzhi2333** [Calamari-OCR/calamari#4]
Is the parameter  --weights WEIGHTS     Load network weights from the given file ?

**Comment by ChWick** [comment]
yes, to restore from a checkpoint just load  either NAME.ckpt or NAME.ckpt.json via the --weights param.

**Comment by christophered** [comment]
@erjinzhi2333 if you were able to continue training, please close this topic.
